src/features/TodoContent/component.tsx

```diff
@@ -37,42 +37,68 @@ const filterTodos = (todos: Todo[], filterStatus: FilterStatus) => {
     });
 };
 
-export const TodoContent = () => {
-  const [updateModalOpen, setUpdateModalOpen] = useState(false);
-  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+type TodoContentProps = {
+  todos: Todo[];
+  filterStatus: FilterStatus;
+};
+
+type TodoContentState = {
+  modalOpen: boolean;
+  selectedTodo: Todo | null;
+};
 
-  const todos = useSelector((state: RootState) => state.todo.todoList);
-  const filterStatus = useSelector(
-    (state: RootState) => state.todo.filterStatus,
-  );
+class Content extends Component<TodoContentProps, TodoContentState> {
+  constructor(props: TodoContentProps) {
+    super(props);
 
-  const filteredTodos = filterTodos(todos, filterStatus);
+    this.state = {
+      modalOpen: false,
+      selectedTodo: null,
+    };
+  }
 
-  const handleEdit = (todo: Todo) => {
-    setSelectedTodo(todo);
-    setUpdateModalOpen(true);
+  handleEdit = (todo: Todo) => {
+    this.setState({ selectedTodo: todo, modalOpen: true });
   };
 
-  return (
-    <Container
-      sx={{
-        backgroundColor: 'lightBlue',
-        borderRadius: '10px',
-        padding: '10px',
-      }}
-    >
-      {filteredTodos.length > 0 ? (
-        filteredTodos.map((todo) => (
-          <TodoItem todo={todo} key={todo.id} onEdit={() => handleEdit(todo)} />
-        ))
-      ) : (
-        <Typography>No tasks found.</Typography>
-      )}
-      <TodoModal
-        modalOpen={updateModalOpen}
-        setModalOpen={setUpdateModalOpen}
-        todo={selectedTodo}
-      />
-    </Container>
-  );
-};
+  render() {
+    const { modalOpen, selectedTodo } = this.state;
+    const { todos, filterStatus } = this.props;
+
+    const filteredTodos = filterTodos(todos, filterStatus);
+
+    return (
+      <Container
+        sx={{
+          backgroundColor: 'lightBlue',
+          borderRadius: '10px',
+          padding: '10px',
```

Comment:
```suggestion
          borderRadius: 10,
          padding: 10,
``` 
Noticed this approach in other places as well. Can choose one approach for consistency?